src/CommandLine/Endpoint.cs

```diff
@@ -92,5 +99,63 @@ public static async Task Unsubscribe(IAmazonSQS sqs, IAmazonSimpleNotificationSe
                 await Topic.Delete(sns, topicArn);
             }
         }
+
+        public static async Task SetPolicy(IAmazonSQS sqs, IAmazonSimpleNotificationService sns, string prefix, string endpointName, IEnumerable<string> eventTypes, bool addAccountCondition, bool addPrefixcondition, IReadOnlyList<string> namespaceConditions)
+        {
+            await Console.Out.WriteLineAsync($"Setting policy on endpoint '{endpointName}'.");
+
+            var policyStatements = new List<PolicyStatement>();
+            var queueUrl = await Queue.GetUrl(sqs, prefix, endpointName);
+            var queueAttributes = await sqs.GetAttributesAsync(queueUrl).ConfigureAwait(false);
+            var queueArn = queueAttributes["QueueArn"];
+
+            foreach (var eventType in eventTypes)
+            {
+                var topicArn = await Topic.Get(sns, prefix, eventType);
+                if (topicArn == null)
+                {
+                    await Console.Out.WriteLineAsync($"No topic detected for event type '{eventType}', please subscribe to the event type first..");
+                    await Console.Out.WriteLineAsync($"Policy on endpoint '{endpointName}' not set.");
+                    return;
+                }
+                policyStatements.Add(new PolicyStatement($"{prefix}{eventType}", topicArn, queueArn));
+            }
+
+            var policy = queueAttributes.ExtractPolicy();
+
+            if (!policy.Update(policyStatements,
+                addAccountCondition, 
+                addPrefixcondition,
+                namespaceConditions,
+                prefix,
+                queueArn))
+            {
+                await Console.Out.WriteLineAsync($"Policy on endpoint '{endpointName}' not set.");
```

Comment:
Maybe we could tone this down a bit to not give the impression something is off. How about something like

```suggestion
                await Console.Out.WriteLineAsync($"No updates needed for policy on endpoint '{endpointName}'.");
```